codalab/lib/bundle_cli.py

```diff
@@ -2589,7 +2589,7 @@ def do_ginfo_command(self, args):
         group = client.fetch('groups', args.group_spec)
 
         members = []
-        # group['owner'] may be None (i.e. for the public group)
+        # group['owner'] may be a falsey null-relationship (i.e. for the public group)
```

Comment:
No, for the JavaScript client it should just be null. Also implementation for JavaScript should be a lot simpler since the scope for updating attributes/relationships will be a lot smaller, at least at first. 

I do want to think about how to make this a little simpler though, the JsonApiClient stuff seems like it's starting to get a bit unwieldy and maybe it can be much simplified with some thought. I'll stew on this for a bit longer.